core/serv/src/extension.rs

```diff
@@ -132,6 +132,7 @@ async fn monitor(extension: Extension, ctx: ExtensionCtx) {
     let _ = futures::future::select(interrupted, restart_loop).await;
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
Consider creating an issue for revisiting all `#[allow(clippy::large_enum_variant)]`.